pkg/cloud/services/elb/loadbalancer_test.go

```diff
@@ -228,6 +228,20 @@ func TestGetAPIServerClassicELBSpec_ControlPlaneLoadBalancer(t *testing.T) {
 				}
 			},
 		},
+		{
+			name: "load balancer config with elb protocol specified",
```

Comment:
```suggestion
			name: "load balancer config with elb health check protocol specified",
```

Reply:
Also needs a test to confirm it is set to SSL by default, but this test will be in the webhooks after doing the changes above.

Reply:
Done